{"uid":"71c65a80aecf71de","name":"test_forceful_restart_without_data_and_transaction_logs","fullName":"tests.integration.ha_tests.test_self_healing#test_forceful_restart_without_data_and_transaction_logs","historyId":"1ab295315f12ae381caee21d2bb9082a","time":{"start":1713925505520,"stop":1713925583750,"duration":78230},"description":"A forceful restart with deleted data and without transaction logs (forced clone).","descriptionHtml":"<p>A forceful restart with deleted data and without transaction logs (forced clone).</p>\n","status":"broken","statusMessage":"Exception: ('Expected command %s to succeed instead it failed: %s', 'ls -1 /var/snap/charmed-postgresql/common/var/lib/postgresql/pg_wal/', 1)","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 849, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 883, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 256, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 182, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 172, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1772, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 195, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/tests/integration/ha_tests/test_self_healing.py\", line 356, in test_forceful_restart_without_data_and_transaction_logs\n    files = await list_wal_files(ops_test, app)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/tests/integration/ha_tests/helpers.py\", line 556, in list_wal_files\n    stdout = await run_command_on_unit(ops_test, unit, command)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/tests/integration/helpers.py\", line 864, in run_command_on_unit\n    raise Exception(\nException: ('Expected command %s to succeed instead it failed: %s', 'ls -1 /var/snap/charmed-postgresql/common/var/lib/postgresql/pg_wal/', 1)","flaky":true,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1713922202341,"stop":1713922202341,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1713922202342,"stop":1713922202342,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1713922202342,"stop":1713922202737,"duration":395},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"primary_start_timeout","time":{"start":1713923688967,"stop":1713923695905,"duration":6938},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1713925501281,"stop":1713925501282,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1713925501281,"stop":1713925501281,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"wal_settings","time":{"start":1713925501282,"stop":1713925505519,"duration":4237},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"wal_settings::finalizer","time":{"start":1713925583758,"stop":1713925616498,"duration":32740},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes::finalizer","time":{"start":1713925616500,"stop":1713925617544,"duration":1044},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1713925617545,"stop":1713925617545,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"primary_start_timeout::finalizer","time":{"start":1713926171692,"stop":1713926171760,"duration":68},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1713926171763,"stop":1713926172308,"duration":545},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1713926172310,"stop":1713926172310,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1713926172310,"stop":1713926172311,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1713926172311,"stop":1713926172311,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"unstable"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"large-429-2764cce77d7db6163c75f7d5"},{"name":"thread","value":"10742-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":5,"skipped":0,"passed":5,"unknown":0,"total":11},"items":[{"uid":"42112ef7d433df9f","reportUrl":"../16//#testresult/42112ef7d433df9f","status":"passed","time":{"start":1713837613186,"stop":1713837793619,"duration":180433}},{"uid":"e8a0d422c8ea002a","reportUrl":"../15//#testresult/e8a0d422c8ea002a","status":"broken","statusDetails":"Exception: ('Expected command %s to succeed instead it failed: %s', 'ls -1 /var/snap/charmed-postgresql/common/var/lib/postgresql/pg_wal/', 1)","time":{"start":1713751309923,"stop":1713751390276,"duration":80353}},{"uid":"e18faa2c2eab7954","reportUrl":"../11//#testresult/e18faa2c2eab7954","status":"passed","time":{"start":1713492087127,"stop":1713492260669,"duration":173542}},{"uid":"c08ea230a8abef0a","reportUrl":"../10//#testresult/c08ea230a8abef0a","status":"passed","time":{"start":1713405521378,"stop":1713405696870,"duration":175492}},{"uid":"a0a9df9470f98a3a","reportUrl":"../9//#testresult/a0a9df9470f98a3a","status":"failed","statusDetails":"AssertionError: Unable to create continuous_writes table","time":{"start":1713320820952,"stop":1713321124350,"duration":303398}},{"uid":"be2541f059ae167","reportUrl":"../8//#testresult/be2541f059ae167","status":"passed","time":{"start":1713232971088,"stop":1713233142935,"duration":171847}},{"uid":"38a4d0c8659cca8d","reportUrl":"../7//#testresult/38a4d0c8659cca8d","status":"passed","time":{"start":1713153378272,"stop":1713153509832,"duration":131560}},{"uid":"e301147dedf2c5ee","reportUrl":"../6//#testresult/e301147dedf2c5ee","status":"broken","statusDetails":"Exception: ('Expected command %s to succeed instead it failed: %s', 'ls -1 /var/snap/charmed-postgresql/common/var/lib/postgresql/pg_wal/', 1)","time":{"start":1713060264329,"stop":1713060344289,"duration":79960}},{"uid":"508c0e44ac7ecbd0","reportUrl":"../4//#testresult/508c0e44ac7ecbd0","status":"broken","statusDetails":"Exception: ('Expected command %s to succeed instead it failed: %s', 'ls -1 /var/snap/charmed-postgresql/common/var/lib/postgresql/pg_wal/', 1)","time":{"start":1712973396966,"stop":1712973476428,"duration":79462}},{"uid":"783f6783b1955f3b","reportUrl":"../2//#testresult/783f6783b1955f3b","status":"broken","statusDetails":"Exception: ('Expected command %s to succeed instead it failed: %s', 'ls -1 /var/snap/charmed-postgresql/common/var/lib/postgresql/pg_wal/', 1)","time":{"start":1712887350309,"stop":1712887434955,"duration":84646}}]},"tags":["unstable","asyncio","group(1)"]},"source":"71c65a80aecf71de.json","parameterValues":[]}